### ISSUE TITLE
Remove manual masking features

### ISSUE THREAD

**Issue by jcohenadad** [shimming-toolbox/fsleyes-plugin-shimming-toolbox#49]
To follow-up on https://github.com/shimming-toolbox/fsleyes-plugin-shimming-toolbox/issues/48#issuecomment-1139641852, I would like us to re-think the relevance of the manual masking feature in ST. 

FSLeyes being an MRI viewer, it offers extremely powerful capabilities for drawing manual masks. Here is an example of a 3D mask, created in only a few seconds:

https://user-images.githubusercontent.com/2482071/170714787-b1994e0d-ffa3-40a2-9868-67259c7a6c1c.mov

I believe this approach should be preferred because:
- It wouldn't be our responsibility to develop/maintain code related to manual masking;
- FLSeyes already provide the feature and it works great;
- It is more user-friendly to create the mask directly on the image, rather than having to find coordinates, enter them manually, check what would should be the appropriate size, etc. 
- The current ST-based manual masking does not easily allow for complex geometries.

**Comment by 4rnaudB** [comment]
Just discovered this feature and it feels like a second mask tab. You would want to replace the existing mask tab in shimming toolbox with this?

**Comment by jcohenadad** [comment]
> You would want to replace the existing mask tab in shimming toolbox with this?

not the tab (because SCT and BET masking are not offered by FSLeyes), but only the 'manual' mask from the combo box

**Comment by po09i** [comment]
I agree, I think adding a Shimming Toolbox masking feature in FSLeyes results in duplication of what is already a well designed feature. I think we should still keep the tab for both SCT and BET.

Note: I think the manual masking features are still relevant for the CLI version since we don't have access to the GUI.

**Comment by jcohenadad** [comment]
> Note: I think the manual masking features are still relevant for the CLI version since we don't have access to the GUI.

Agreed, we can leave it, but I would not spend too much time adding features to this CLI-based manual masking. I think that most users will use ST from the GUI
